lib/request.js

```diff
@@ -845,10 +855,14 @@ function patch(Request) {
      * @function closed
      * @returns {Boolean} is closed
      */
-    Request.prototype.closed = function closed() {
+    function closed() {
         var self = this;
-        return self.connectionState() === 'close';
-    };
+        if (this._readableState && 'close' in this._readableState) {
```

Comment:
typo

```suggestion
        if (this._readableState && 'closed' in this._readableState) {
```

and tests are failing with typo fixed. I'm trying to figure out if there's a way to do this without monkey patching.